go.mod

```diff
@@ -6,6 +6,8 @@ replace github.com/hashicorp/vault/api => ./api
 
 replace github.com/hashicorp/vault/sdk => ./sdk
 
+replace github.com/hashicorp/raft-boltdb/v2 => ../raft-boltdb/v2
```

Comment:
Will this not take the reference to outside of the repo? I guess this was only intended to be part of testing?

Reply:
Yeah, sorry for any confusion. I'm still working on this PR. I tend to push up whatever I've worked on at the end of the day. This won't be in the final version.